core/src/test/java/de/eldoria/bloodnight/editor/EditorPayloadTest.java

```diff
@@ -87,7 +87,7 @@ void asJson() throws JsonProcessingException {
                 ValueModifier.MULTIPLY,
                 2);
 
-        Extension extension = new Extension(ExtensionType.CARRIER, EntityType.SPIDER);
+        Extension extension = new Extension(ExtensionType.CARRIER, EntityType.SPIDER, null);
```

Comment:
Please create an empty constant for an empty equipment object and pass it here.

Reply:
👍